it/controllers/benefits/income/IncomeTaxBenefitsAmountControllerISpec.scala

```diff
@@ -0,0 +1,574 @@
+    val expectedErrorNoEntry = "Enter the amount of Income Tax paid by your client’s employer"
+    val expectedErrorIncorrectFormat = "Enter the amount of Income Tax paid by your client’s employer in the correct format"
+    val expectedErrorOverMaximum = "The Income Tax paid by your client’s employer must be less than £100,000,000,000"
+  }
+
+  object ExpectedAgentCY extends SpecificExpectedResults {
+    val expectedTitle = "How much of your client’s Income Tax did their employer pay?"
+    val expectedHeading = "How much of your client’s Income Tax did their employer pay?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val expectedErrorNoEntry = "Enter the amount of Income Tax paid by your client’s employer"
+    val expectedErrorIncorrectFormat = "Enter the amount of Income Tax paid by your client’s employer in the correct format"
+    val expectedErrorOverMaximum = "The Income Tax paid by your client’s employer must be less than £100,000,000,000"
+  }
+
+  val userScenarios: Seq[UserScenario[CommonExpectedResults, SpecificExpectedResults]] = {
+    Seq(UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN)),
+      UserScenario(isWelsh = false, isAgent = true, CommonExpectedEN, Some(ExpectedAgentEN)),
+      UserScenario(isWelsh = true, isAgent = false, CommonExpectedCY, Some(ExpectedIndividualCY)),
+      UserScenario(isWelsh = true, isAgent = true, CommonExpectedCY, Some(ExpectedAgentCY)))
+  }
+
+  ".show" should {
+
+    userScenarios.foreach { user =>
+      import Selectors._
+      import user.commonExpectedResults._
+
+      s"language is ${welshTest(user.isWelsh)} and request is from an ${agentTest(user.isAgent)}" should {
+        "render the how much of your client’s Income Tax did their employer pay?' page with an empty amount field" which {
+          implicit lazy val result: WSResponse = {
+            authoriseAgentOrIndividual(user.isAgent)
+            dropEmploymentDB()
+            userDataStub(userData(fullEmploymentsModel()), nino, taxYearEOY)
+            insertCyaData(employmentUserData(hasPriorBenefits = true, cyaModel(
+              Some(benefits(fullIncomeTaxOrIncurredCostsModel.copy(incomeTaxPaidByDirector = None))))), userRequest)
+            urlGet(pageUrl(taxYearEOY), welsh = user.isWelsh, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+          s"has an OK($OK) status" in {
+            result.status shouldBe OK
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(expectedCaption, captionSelector)
+          textOnPageCheck(enterTotalText, paragraphSelector(2))
+          textOnPageCheck(expectedHintText, hintTextSelector)
+          elementNotOnPageCheck(paragraphSelector(3))
```

Comment:
this still isnt right, you need to check that the "if this is not XXX" is not on the page. This is checking for a p3 which will never be on the page.